### ISSUE TITLE
No upcoming events

### ISSUE THREAD

**Issue by MrCupola** [galaktor/MMM-DublinRTPI#1]
The only timetable working is the one for Dart, also using the configuration given in the example, no upcoming events for the bus neither for the Luas...
Could be a temporary problem with Dublinked RTPI API ?

**Comment by MrCupola** [comment]
I don't understand what's the cause, but some bus stop doesn't work and a lot of Luas stop neither, others just work fine.
Tried to use filters to display only some specific routes or some specific Luas direction, but they aren't working, none of these:
routes: ["40"]
directions: ["south"]
directions: ["Inbound"]
destination: ["LUAS Tallaght"]

**Comment by Fitz47** [comment]
Was there a solution to this issue? I'm seeing the same thing but with every LUAS stopid I've tried.

**Comment by MrCupola** [comment]
I wasn't able to solve the problem, my workaround has been to select the previous bus stop and to display luas timetable without filters

**Comment by darts** [comment]
I looked for a source of real time Dublin Bus data and found this under FAQs "The National Transport Authority intends to make an API available when the multi-operator web service is launched." here : https://www.dublinbus.ie/RTPI/RTPI-Help-Page/#8 . This module pulls from a link that doesn't contain public data anymore (https://data.dublinked.ie/cgi-bin/rtpi/realtimebusinformation?stopid=[stopID]). 
TLDR: as far as I can see, real time Dublin Bus data is not available through an API.

**Comment by MrCupola** [comment]
Unfortunately they blocked the 3rd party access to the API few weeks ago due to an excess amount of access that caused the server to crash

**Comment by MrCupola** [comment]
The API is working again

**Comment by darts** [comment]
Thanks for the heads up

**Comment by ghost** [comment]
Hi, is this Mod this functioning? I did a install but not getting any timetable displaying. 
just a Loading ...

**Comment by ThunderWolf19** [comment]
> Hi, is this Mod this functioning? I did a install but not getting any timetable displaying.
> just a Loading ...

I'm having the same issue with an ERR! code ELIFECYCLE
